devices.js

```diff
@@ -17524,6 +17524,28 @@ const devices = [
         extend: preset.light_onoff_brightness_colortemp({colorTempRange: [200, 370]}),
     },
 
+    // Matcall BV
+    {
+        zigbeeModel: ['ZG 401224'],
+        model: 'ZG 401224',
+        vendor: 'Matcall bv',
+        description: 'Matcall LED Dimmer driver',
+        exposes: [e.light_brightness()],
```

Comment:
For both `exposes` `fromZigbee` and `toZigbee` can be replaced with `extend: preset.light_onoff_brightness(),`

Reply:
OK will do, although I cannot test that myself